client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -119,8 +119,12 @@
 	padding: 80px 20px 10px;
 	display: none;
 
-	button {
+	.button.navigation-link {
```

Comment:
I've changed the selector because it was insufficient when it comes to CSS specificity